tests/integration/ceph_base_object_test.go

```diff
@@ -142,6 +92,74 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", namespace, 1, "Running"))
 	logger.Infof("Ceph MGRs are running alright")
 
+	/* Testing creation/deletion of objects using Object Bucket Claim */
+	logger.Infof("Step 3 : Create Object Bucket Claim with reclaim policy delete")
+
+	cobErr := helper.BucketClient.CreateBucketStorageClass(namespace, storeName, storageClassName, "Delete", region)
+	require.Nil(s.T(), cobErr)
+	cobcErr := helper.BucketClient.CreateObc(obcName, storageClassName, bucketname, true)
+	require.Nil(s.T(), cobcErr)
+
+	logger.Infof("Checking to see if the obc,secret,cm has been created")
+	for i = 0; i < 4 && helper.BucketClient.ObcExists(obcName) == false; i++ {
```

Comment:
The obc is created above. It should appear almost instantly unless the k8s api server is very busy. I guess past rook testing experience suggests that there can be a delay in creating k8s resources. I just want to make sure you know that the creation of the obc is independent of rook, operators, the bucket library, etc. It is a simple k8s CR create and if there is a delay that may indicate a busy api server. cc @travisn

Reply:
Oh right, we shouldn't need the retry for the OBC. We should only need the retry in the case where the operator is creating a resource in the reconcile loop. In this case, the bucket could be delayed, but not the OBC.